optimize/backend/src/it/java/io/camunda/optimize/AbstractCCSMIT.java

```diff
@@ -100,14 +93,22 @@ protected static boolean isZeebeVersion87OrLater() {
     final Pattern zeebeVersionPattern = Pattern.compile("8.([7-9]|\\d{2,})");
     return zeebeVersionPattern
         .matcher(IntegrationTestConfigurationUtil.getZeebeDockerVersion())
-        .matches();
+        .matches()
+        || isZeebeVersionSnapshot();
   }
 
+  // After the 8.7.0 release, isZeebeVersion87OrLater() needs to call this method instead of
+  // isZeebeVersionSnapshot(), since latest is until then an 8.6.x release
```

Comment:
So does this become a change then that needs making each time a new minor release is made? Did I understand this right?

Reply:
Not each time. This concrete method just needs to be changed once, since we only have this situation while `latest` is not a 8.7.0 version. This issue will however re-occur if we need to make checks for e.g. `is 880 or later`, because then at that time latest will be `8.7.x`. We could consider whether we actually have any value in testing `latest`, since that is sort of covered by testing the `8.6.0`. Then we don't need to make this differentiaton (see screenshot). What do you think @RomanJRW, shall I just remove `latest` from the pipeline and also the entire logic for it in the IT?
<img width="398" alt="image" src="https://github.com/user-attachments/assets/f513be8a-8cb6-4d80-8754-bc11f0a66b89" />

Reply:
Yeah I would say it's easier and cleaner to remove latest testing completely. We don't get much value from it, like you say 👍